app/controllers/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class UsersController < ApplicationController
+  def initialize
+    super
+    @users = User.find(1, 2)
+  end
+
+  def show; end
```

Comment:
- [ ] [optional] @KareemWilson kindly add index method, its good practice to have a method that corresponds with the view eg an index page with index method